params/astria_config.go

```diff
@@ -146,6 +169,42 @@ func NewAstriaForks(forks map[string]AstriaForkConfig) (*AstriaForks, error) {
 			orderedForks[i].SnapshotChecksum = currentFork.SnapshotChecksum
 		}
 
+		if currentFork.AppSpecificOrdering != nil {
+			if len(currentFork.AppSpecificOrdering) == 0 {
+				log.Debug("currentFork.AppSpecificOrdering before", "ordering", fmt.Sprintf("%v", currentFork.AppSpecificOrdering))
+			} else {
+				log.Debug("currentFork.AppSpecificOrdering before", "ordering", fmt.Sprintf("%v", currentFork.AppSpecificOrdering))
+			}
+		} else {
+			log.Debug("currentFork.AppSpecificOrdering before", "ordering", "nil")
+		}
+		// Apply ordering rules to the current fork
+		if currentFork.AppSpecificOrdering == nil && i > 0 {
+			// Carry over ordering from previous fork
+			orderedForks[i].AppSpecificOrdering = orderedForks[i-1].AppSpecificOrdering
+		} else if currentFork.AppSpecificOrdering != nil && len(currentFork.AppSpecificOrdering) == 0 {
+			// Explicitly reset ordering to empty (not nil)
+			orderedForks[i].AppSpecificOrdering = []AstriaTransactionType{}
+		} else if currentFork.AppSpecificOrdering != nil {
+			// Apply set ordering rules to the current fork
+			orderedForks[i].AppSpecificOrdering = nil
+			for _, transactionType := range currentFork.AppSpecificOrdering {
+				if _, ok := AstriaTransactionTypeMap[transactionType]; !ok {
+					return nil, fmt.Errorf("invalid transaction type: %s", transactionType)
+				}
+				orderedForks[i].AppSpecificOrdering = append(orderedForks[i].AppSpecificOrdering, AstriaTransactionTypeMap[transactionType])
+			}
+		}
+		if orderedForks[i].AppSpecificOrdering != nil {
+			if len(orderedForks[i].AppSpecificOrdering) == 0 {
+				log.Debug("orderedForks.AppSpecificOrdering after", "index", i, "ordering", "[]")
+			} else {
+				log.Debug("orderedForks.AppSpecificOrdering after", "index", i, "ordering", fmt.Sprintf("%v", orderedForks[i].AppSpecificOrdering))
+			}
```

Comment:
```suggestion
		    log.Debug("orderedForks.AppSpecificOrdering after", "index", i, "ordering", fmt.Sprintf("%v", orderedForks[i].AppSpecificOrdering))
```

I believe this will print the same thing